client/index.jsx

```diff
@@ -125,7 +125,9 @@ Promise.all([rootElemPromise])
       const config = await configPromise
       serverConfig.setValue(config)
     } catch (err) {
-      // TODO(2Pac): Do something with the error
+      // Ignoring the error here shouldn't be that big of a deal since the config is usually cached
```

Comment:
I'd probably just log the error at least (as a warning probably)

Generally wouldn't want to swallow these errors completely as it can make it hard to spot weird issues